naming_conventions.md

```diff
@@ -0,0 +1,73 @@
+# Repositories and packages naming conventions
+
+*All new public repositories and packages should follow naming conventions, we don't want to rename existing projects. Provided examples are for visualizing all currently available repositories with the proposed naming conventions.*
+
+## Repositories
+
+*The main idea in the proposed naming convention is to provide easy readability and information to the developers. Moreover, it will help us make the organization consistent. In most cases, the developer is interested in the repositories belonging to a specific product, therefore, the repository name starts with the product name. Then developer is looking for the specific project for his/her needs for the specific platform. Therefore `<project-name>-<language/platform>` suffix.*
+
+---
+
+### Is the repository related to the company's product?
+
+#### Format: `<product>-<project-name>-<language/platform>`
+
+- `<product>` - mandatory, related product
+  - *Examples: `fingerprintjs`, `fingerprintjs-pro`, `botd`*
+- `<project-name>` - mandatory, descriptive name of the project
+  - *Examples: `server-api-sdk`, `agent`, `client`, `integrations`*
+- `<language/platform>` - optional, use when makes sense or it's plausible there will be available the same functionality for different language/platform in the future. Mostly apply to agents, wrappers, or SDKs.
+  - *Examples: `javascript`, `node`, `react`, `python`*
+
+#### Examples
+
+- `fingerprintjs-agent` or `fingerprintjs-client`
+- `botd-agent` or `botd-client`
+- `botd-integrations`
+- `fingerprint-android-client` or `fingerprintjs-android-client`
+---
+
+### Is the repository related to the content, blog post, article, or video? Is it just an example without any supporting content?
+
+#### Format: `blog-<name-of-blog-or example>-<type>`
+
+- `blog` - optional, mandatory if repository supports article and is not explanatory enough without supporting content
+- `<name-of-blog-or-example>` - mandatory, should be descriptive or aligned with the blogpost name, if related to product, should start with product name
+  - Examples: `adblocker-fingerprinting`, `audio-fingerprinting`, `fingerprintjs-react`
+- `<type>` - optional, might be ommited if not used for blog
```

Comment:
In my opinion such suffixes should be mandatory.
If I just see `adblocker-fingerprinting`, `audio-fingerprinting`, `fingerprintjs-react` names it's not clear for me what is it? Some modules for fingerprintjs project? Standalone projects for specific use case? Fingerprintjs client for React? With `-example`, `-demo` it will be clear without any context.

Reply:
ok, let's try it with mandatory, we can adjust it according to our needs in the future, ty